apps/docs/content/components/Heading.mdx

```diff
@@ -24,32 +24,43 @@ import {Heading} from '@primer/react-brand'
 
 ### Scale
 
+Heading sizes range from 1-6 and map to positions 900-400 on the Brand type scale respectively.
+
 ```jsx live
 <>
-  <Heading as="h1">Heading 1</Heading>
-  <Heading as="h2">Heading 2</Heading>
-  <Heading as="h3">Heading 3</Heading>
-  <Heading as="h4">Heading 4</Heading>
-  <Heading as="h5">Heading 5</Heading>
-  <Heading as="h6">Heading 6</Heading>
+  <Heading as="h1">1: 72px</Heading>
```

Comment:
@ashygee - changes we discussed have been applied. FYI, these are desktop only px sizes, which change at narrow and regular viewports.

Reply:
We might want to note that these are desktop sizes and link to a page in the docs about our type scale but that can be done after this.